python/benchmark_db.py

```diff
@@ -15,26 +16,33 @@ def benchmark(
     require_images: bool = typer.Option(True, help="Request the original images"),
     require_embeddings: bool = typer.Option(False, help="Request embeddings"),
     test_masks: bool = typer.Option(True, help="Test masks"),
-    test_latents: bool = typer.Option(True, help="Test latents"),
 ):
     print(f"Running benchmark for {source} - {limit} samples")
-
-    # Get a generic client config
-    client_config = datago.GetDatagoConfig()
-    client_config.ImageConfig.CropAndResize = crop_and_resize
-
-    # Specify the source parameters as you see fit
-    source_config = datago.GetSourceDBConfig()
-    source_config.Sources = source
-    source_config.RequireImages = require_images
-    source_config.RequireEmbeddings = require_embeddings
-    source_config.HasMasks = "segmentation_mask" if test_masks else ""
-    source_config.HasLatents = "caption_latent_t5xxl" if test_latents else ""
-
-    # Get a new client instance, happy benchmarking
-    client_config.SourceConfig = source_config
-    client = datago.GetClient(client_config)
-
+    client_config = {
```

Comment:
previous implementation was buggy, in that the update to the config object in the python space wouldn't be reflected in go. This json/dict approach is readable and not broken